src/query/anonymization.c

```diff
@@ -644,9 +644,12 @@ static void prepare_bucket_seeds(Query *query, AnonymizationContext *anon_contex
   ListCell *cell = NULL;
   foreach (cell, filtering_consts)
   {
-    Const *label = castNode(Const, unwrap_cast(lfirst(cell)));
-    hash_t label_hash = hash_label(label->consttype, label->constvalue, label->constisnull);
-    anon_context->base_labels_hash_set = hash_set_add(anon_context->base_labels_hash_set, label_hash);
+    Oid type;
+    Datum value;
+    bool isnull;
+    get_stable_expression_value(unwrap_cast(lfirst(cell)), bound_params, &type, &value, &isnull);
```

Comment:
Not related to this PR, but "stable expression" sounds a bit weird to me in this context.
I think "static expression" would have worked better.

Reply:
we had some discussion with Edon on this. "stable expression" is coming from PostgreSQL, and we opted to adopt that term (the previous term, and what remains in the public-facing copy was "simple constant").

Reply:
does this explanation make "stable expression" work for you?

Reply:
Sorry, I didn't realize you were expecting a reply from me here. Yes, I'm fine with it as is.
I have already approved, so this wasn't a blocker. Please go ahead with the merge.